Makefile

```
@@ -75,12 +75,20 @@ install:
 	mkdir --parents $(SERVICEPATH)
 	cp --force $(service) $(SERVICEPATH)
 	@echo ""
-
+	
 	@echo "# Copying default configuration file to $(CONFIGPATH)/$(config)"
 	mkdir --parents $(CONFIGPATH)
 	-cp --no-clobber $(config) $(CONFIGPATH)
 	@echo ""
 
+	@echo "# Do you want to add all currently connected keyboards to your configuration? (y/N)"
+	@read -r answer; \
+	if [ "$$answer" = "y" ] || [ "$$answer" = "Y" ]; then \
+		echo "\nUpdating configuration..."; \
```

This should be
```suggestion
		echo ""; \
		echo "Updating configuration..."; \
```
_Matches other new lines in this file_

or

```suggestion
		echo -e "\nUpdating configuration..."; \
```

The \n showed up in the output when I ran it.

---

Checked! I didn't catch print convention. I commited the first version that using `echo ""`. Thanks!